app/controllers/admin/items_controller.rb

```diff
@@ -26,7 +26,7 @@ def edit
 
   def update
     @item = Item.find(params[:id])
-
+    @item.update(image_tag: nil) if item_params[:image]
```

Comment:
@valerietrudell @CjMoore let @meyerhoferc or I know if you have questions on what this is doing/why we have it! Also -- thoughts on refactoring this into the Item model? That's how we originally thought to do it!